### PR TITLE
Implement `format_serde_error`

### PR DESCRIPTION
**Pull request by hendrikmaus**
This changeset implement the `format_serde_error` crate to make configuration errors readable for humans, e.g.: 
image

## To do

- [x] switch to the latest release of `format_serde_error` once https://github.com/AlexanderThaller/format_serde_error/pull/15 was released
- [x] discuss whether we want to continue to support json, toml and yaml

Support for TOML and JSON was dropped.

### CODE REVIEW

**Comment by hendrikmaus** [comment]
@AlexanderThaller @Kenec I would like to get your stand on a proposal: "Let's *only* support YAML as configuration format."

**Comment by hendrikmaus** [comment]
Blocked until the next `format_serde_error` release.

File: README.md [hunk 1]
```diff
@@ -3,34 +3,33 @@
...
       └── nginx-chart
 ```
 
 ## Configuration
 
-Configuration can be provided as TOML, YAML or JSON - please also **see the [examples](tests/data)**.
+Configuration can be provided as YAML format.
 
-Please mind that all paths are evaluated relative to the working directory you call `helm-templexer` from.
+> All paths are evaluated relative to the configuration ile during execution.
```
Comment (Kenec) [inline README.md:32]:
```suggestion
> All paths are evaluated relative to the configuration file during execution.
```

Comment (hendrikmaus) [reply README.md:32]: Good catch.

**Review by Kenec — APPROVED** [review]

**Pull request merged**